# command message

# NED commands

float32 roll         # radians
float32 pitch        # radians
float32 yaw_rate     # radians
float32 altitude     # meters
bool reset